add rn domain change (#8016)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

Release notes are grouped by month for both multi-tenant and virtual private cloud (VPC) environments.

## October 2025

> - **Behavior change**: dbt platform [access URLs](/docs/cloud/about-cloud/access-regions-ip-addresses) for accounts in the US multi-tenant (US MT) region are transitioning from `cloud.getdbt.com` to dedicated domains on `dbt.com` (for example, `us1.dbt.com`). Users will be automatically redirected, which means no action is required. EMEA and APAC MT accounts are not impacted by this change and will be updated by the end of November 2025.

Reviewer: ```suggestion
- **Behavior change**: <Constant name="dbt_platform" /> [access URLs](/docs/cloud/about-cloud/access-regions-ip-addresses) for accounts in the US multi-tenant (US MT) region are transitioning from `cloud.getdbt.com` to dedicated domains on `dbt.com` (for example, `us1.dbt.com`). Users will be automatically redirected, which means no action is required. EMEA and APAC MT accounts are not impacted by this change and will be updated by the end of November 2025.
```

Author: oh nice, i didnt realize it was a constant. thank you!

>   Organization that use network allow-listing should add `YOUR_ACCESS_URL.dbt.com` to their allow list (for example, if your access URL is `us1.dbt.com`, add `us1.dbt.com` to your allow list). 

Reviewer: ```suggestion
  Organizations that use network allow-listing should add `YOUR_ACCESS_URL.dbt.com` to their allow list (for example, if your access URL is `us1.dbt.com`, add `us1.dbt.com` to your allow list). 
```

>   All OAuth, Git, and public API integrations will continue to work with the previous domain. View the updated access URL in dbt platform's **Account settings** page.

Reviewer: ```suggestion
  All OAuth, Git, and public API integrations will continue to work with the previous domain. View the updated access URL in <Constant name="dbt_platform" />'s **Account settings** page.
```
